src/screens/FeedScreen/RenderList.js

```diff
@@ -120,7 +210,11 @@ const RenderListFeed = (props) => {
           source={source}
           headerStyle={styles.mh9}
           showAnonymousOption={showAnonymousOption}
-          onHeaderOptionClicked={onHeaderOptionClicked}
+          onDeletePost={onDeletePost}
+          isShowDelete={isShowDelete}
+          isSelf={isSelf}
+          isFollow={item?.is_following_target}
+          onPressFollUnFoll={(value) => handleFollowUnfollow(value, false)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New props have been added to the Header component. Ensure that these props are being passed correctly and are used as intended within the Header component.
```diff
- onHeaderOptionClicked={onHeaderOptionClicked}

+ onDeletePost={onDeletePost}
+ isShowDelete={isShowDelete}
+ isSelf={isSelf}
+ isFollow={item?.is_following_target}
+ onPressFollUnFoll={(value) => handleFollowUnfollow(value, false)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->